### ISSUE TITLE
Polygon gaps (color stripes) on rail tracks in Densha de GO! 64 (English fan translation)

### ISSUE THREAD

**Issue by griever-gf** [gonetz/GLideN64#2018]
See screenshots for issue - GLideN64 on the left, software renderer (Angrylions RPD Plus) on the right.
![pan1](https://user-images.githubusercontent.com/1813367/53691129-bf8c4b80-3d88-11e9-8e1d-cbf9150c5e55.png)
![pan2](https://user-images.githubusercontent.com/1813367/53691130-c4e99600-3d88-11e9-8487-60375bc72847.png)
![pan3](https://user-images.githubusercontent.com/1813367/53691132-ca46e080-3d88-11e9-8a18-4b49aa91effb.png)

**Comment by oddMLan** [comment]
Duplicate of #1731

**Comment by gonetz** [comment]
Yes, this problem most likely has the same origin as #1731 and can be tagged with the same tag.
